### PR TITLE
feat: :sparkles: Add feasibility to add `textDirection` of `title` and `description`.

### PR DESCRIPTION
**Pull request by dhvanit-v-simformsolutions**
:sparkles: (https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/issues/148) Add feasibility to add `textDirection` of `title` and `description`.

# Description

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/SimformSolutionsPvtLtd/flutter_showcaseview/blob/master/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org

### CODE REVIEW

File: lib/src/showcase.dart [hunk 1]
```diff
@@ -309,6 +317,8 @@ class Showcase extends StatefulWidget {
     this.onTargetDoubleTap,
     this.disableDefaultTargetGestures = false,
     this.tooltipPosition,
+    this.titleTextDirection,
```
Comment (ujas-m-simformsolutions) [inline lib/src/showcase.dart:320]: Text direction will only be used in tool-tip not in custom widget so don't pass it in constructor. Make it null after constructor initialization.

File: lib/src/showcase.dart [hunk 2]
```diff
@@ -309,6 +317,8 @@ class Showcase extends StatefulWidget {
...
+    this.descriptionTextDirection,
```
Comment (ujas-m-simformsolutions) [inline lib/src/showcase.dart:321]: ^^

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+## (Unreleased)
```
Comment (ujas-m-simformsolutions) [inline CHANGELOG.md:1]: Add version number and do update pubspec and readme

**Review by ujas-m-simformsolutions — CHANGES_REQUESTED** [review]

**Review by ujas-m-simformsolutions — APPROVED** [review]
lgtm

File: README.md [hunk 4]
```diff
@@ -182,6 +182,8 @@ WidgetsBinding.instance.addPostFrameCallback((_) =>
 | tooltipPosition              | TooltipPosition? |                                                  | Defines vertical position of tooltip respective to Target widget                                   | ✅          | ✅                |
 | titlePadding                 | EdgeInsets?      | EdgeInsets.zero                                  | Padding to title                                                                                   | ✅          |                   |
 | descriptionPadding           | EdgeInsets?      | EdgeInsets.zero                                  | Padding to description                                                                             | ✅          |                   |
+| titleTextDirection           | TextDirection?      |                                   | Give textDirection to title                                                                             | ✅          |                   |
+| descriptionPadding           | TextDirection?      |                                   | Give textDirection to description                                                                             | ✅          |                   |
```
Comment (vatsaltanna-simformsolutions) [inline README.md:186]: descriptionPadding -> descriptionTextDirection

**Pull request merged**